docs/src/pages/docs/crocks/Result.md

```diff
@@ -0,0 +1,1422 @@
+---
+title: "Result"
+description: "Result Crock"
+layout: "guide"
+functions: ["trycatch", "eithertoresult", "firsttoresult", "lasttoresult", "maybetoresult"]
+weight: 130
+---
+
+```haskell
+Result e a = Err e | Ok a
+```
+
+Result is a Sum Type a step above [`Maybe`][maybe]. With a [`Maybe`][maybe] the
```

Comment:
It should really be compared with `Either` instead of `Maybe`. With the difference being that you can accumulate `Err`s with `ap` and `alt` when using `Err`s of like `Semigroup`s. Should be mentioned in the body and replace the last code example with a simple example showing that accumulation with either `alt` or `ap`. Does not need to be real world, can save the real world for the method examples for each.